### DISCUSSION TITLE
Osc52 not working in Code Server

### DISCUSSION THREAD

**Discussion by ohjoonhee** [blinksh/blink#1728] [Support]
I’m using code-server with blink, and enjoying the experience. 
I mainly use tmux as my code-server profile, but osc52 doesn’t work in that way. 
However, osc52 works when I’m in ssh session. 
Can you please help me solve the problem?
It’s the one last problem left for my perfect env.

Problem reproduction:
1. Connect to code-server in blink (in my case, ```$ code https://my-domain.com``` my-domain.com directs to the server)
2. In code-server integrated terminal, open tmux session (```$ tmux new -s anysession```)
3. Copy some text by following steps [Tips&Tricks](https://docs.blink.sh/basics/tips-and-tricks)
4. Nothing happens

Working situation:
1. Connect to the server with ssh (```$ ssh my-server-host```)
2. In ssh session, open tmux session
3. Copy some text by the following steps
4. Selected texts are copied to the local(iPad) clipboard.  

Any suggestions will be appreciated, Thanks.

**Comment by carloscabanero** [comment]
Hi! So this falls into VS Code territory, as they are the ones handling the sequence and the clipboard. My suggestion is to connect to the Codespace through Tailscale, that way you can use Blink instead of the VS Code terminal.

Hope this helps!
